_posts/partners-hidora

```diff
@@ -0,0 +1,96 @@
+# Partner Highlight: Enjoy OpenSearch one click installation with Hidora
+
+Who we are
+==========
+
+[Hidora](https://hidora.io) is an expert in providing custom solutions for all sizes of businesses. Hidora Cloud Hosting Solutions are just what you need if you want an efficient, secure and scalable cloud hosting solution that is tailored to your needs. We offer different plans that are scalable according to your specific requirements so you can find one that suits you best. Our PaaS help developers to host their applications in few minutes with DevOps philosophy.
+
+Why choosing Hidora for your OpenSearch hosting ?
+=================================================
+
+With our partner Jelastic, we have integrated the deployment of the OpenSearch stack in every detail. We are currently able to offer you version 1.2.3. Also, redeploy functionality provides a simple update process. You can scale your OpenSearch cluster, extend the storage, add nodes in few seconds.
+
+  
+
+One click installation with the Topology wizard !
+-------------------------------------------------
+
+Take advantage of our customization interface to define the architecture of your cluster:
+
+![](https://image.png)
+
+  
+
+*   Activate or not OpenSearch-Dashboard and Logstash
+*   Define how many nodes will be create on OpenSearch cluster
+*   Assign public IPv4 or IPv6 or manage access through our Shared Load Balancer (SLB)
+*   Scale nodes resources by managing in real time Cloudlet assignation
+
+  
+
+After being created, the OpenSearch node is available on port _9200_ for internal communication between nodes of the environment and port _4848_ for external services through the shared load balancer. The HTTP basic auth is used in both cases (password can be reset using the **Reset Password** button).
+
+![](https:///image.png)
```

Comment:
- Can you merge this with #633 so we can see your photos in place?
- Each image needs a `{: .img-fluid}` at the end of the line. That will make it work on mobile